scripts/contractInteraction/loan_tokens.py

```diff
@@ -855,4 +855,80 @@ def replaceLoanTokenSettingsLowerAdmin():
     loanTokenLogicBeaconWrbtc = Contract.from_abi("LoanTokenLogicBeacon", address=conf.contracts['LoanTokenLogicBeaconWrbtc'], abi=LoanTokenLogicBeacon.abi, owner=conf.acct)
     print("Registering Loan Protocol Settings Module to LoanTOkenLogicBeaconWrbtc")
     data = loanTokenLogicBeaconWrbtc.registerLoanTokenModule.encode_input(loanTokenSettingsLowerAdmin.address)
-    sendWithMultisig(conf.contracts['multisig'], loanTokenLogicBeaconWrbtc.address, data, conf.acct)
\ No newline at end of file
+    sendWithMultisig(conf.contracts['multisig'], loanTokenLogicBeaconWrbtc.address, data, conf.acct)
+
+def transferBeaconOwnershipToGovernance():
+    # transfer beacon LM
+    print("Transferring beacon LM ownserhip to: ", conf.contracts['TimelockOwner'])
```

Comment:
```suggestion
    print("Transferring beacon LM ownserhip to TimelockOwner: ", conf.contracts['TimelockOwner'])
```
pls consider adding the contracts names in all outputs for clarity